vite.config.ts

```diff
@@ -21,7 +23,8 @@ export const configs = {
         }),
       ],
     }),
+    eslintPlugin({ fix: true, cache: true, failOnWarning: true }),
```

Comment:
Ya it'll look for any valid normal `.eslintrc.*` whether `.json` `.ts` or `.js` any that are supported by modern ESLint. in this case our `.json`. I don't see the need for explicitly ecluding `.js` extensions as this will only be touching our source files and if we find some weird case to use `.js` besides for confiig files then it should be linted. and else, no real need to explicitly blacklist.